pages/docs/tutorials/configure-asyncapi-kafka.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Configuring Schemas for Kafka messages using AsyncAPI
+description: In this tutorial, you'll learn how to configure an AsyncAPI document for Kafka messages.
+weight: 80
+---
+
+## Introduction
```

Comment:
```suggestion

```

we usually don't use the "introduction" sub-section/heading, the content at the beginning of the document is often seen as the introduction section, does that make sense?

Reply:
We have actually followed a similar structure in all our previous tutorials, so I think it's a good idea to keep the `Introduction` heading.